backend/Recodo.Common/Dtos/EnumReaction.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Recodo.Common.Dtos
```

Comment:
```suggestion
namespace Recodo.Common.Enums
```
And change the folder. It's not a DTO